telegraf/map.jinja

```diff
@@ -3,7 +3,7 @@
 
 {% set oscodename = salt['grains.filter_by'](oscodename_map, grain='oscodename') %}
 
-{% if not grains['os'] == 'FreeBSD' %}
+{% if not grains['os'] == 'FreeBSD' and not grains['os_family'] == 'RedHat' %}
```

Comment:
Why don't we use only for ubuntu, instead of listing all the ones we _don't want_?

Reply:
Fixed to use just Debian OS Family